apps/emqx/src/emqx_schema.erl

```diff
@@ -3570,6 +3582,18 @@ mqtt_general() ->
                     desc => ?DESC(mqtt_shared_subscription_strategy)
                 }
             )},
+        {"shared_subscription_initial_sticky_pick",
+            sc(
+                hoconsc:enum([
+                    random,
+                    hash_topic,
+                    hash_clientid
+                ]),
+                #{
+                    default => random,
+                    desc => ?DESC(mqtt_shared_subscription_initial_sticky_pick)
+                }
+            )},
```

Comment:
the configs in this section are deprecated and only kept for backward compatibility, please do not add new ones.

Reply:
right, i looked at the wrong lines when reviewing this.